x-pack/plugins/aiops/public/components/search_panel/search_panel.tsx

```diff
@@ -40,15 +39,13 @@ export const SearchPanel: FC<Props> = ({
   searchQueryLanguage,
   setSearchParams,
 }) => {
-  const { unifiedSearch } = getPluginsStart();
-  const { SearchBar } = unifiedSearch?.ui;
   const {
-    services: {
-      uiSettings,
-      notifications: { toasts },
-      data: { query: queryManager },
-    },
-  } = useAiOpsKibana();
+    uiSettings,
+    unifiedSearch,
```

Comment:
Looks like this could just be:
```suggestion
     unifiedSearch: {
       ui: { SearchBar },
     },
```

Reply:
Updated in [9f1a33c](https://github.com/elastic/kibana/pull/140005/commits/9f1a33cf5f2c266f448dc64c33afd4798321bf18).